### DISCUSSION TITLE
Usability - Erfassen/Bearbeiten - neues Objekt

### DISCUSSION THREAD

**Discussion by marko-jer** [e-netze/webgis-community#184] [General]
Bei der Auswahl des Editierthemas ist es nicht ganz intuitiv, dass man auf das Thema Klicken muss, obwohl davor schon eine Check-Box mit Hakerl steht.
Es schaut so aus, als ob das Thema schon ausgewählt wäre. weil erst dann erscheint "Neues Objekt"
(Wenigsten beim ersten Mal sucht man "Neues Objekt" …, in weiterer Folge ist es dann schon klar.)

![image](https://github.com/gis-eni/webgis-forum/assets/1842500/c6578e99-69b0-48ef-97be-59f849b9e836)
![image](https://github.com/gis-eni/webgis-forum/assets/1842500/94536edf-2b2b-43be-9395-3c05439d99ea)

**Comment by gis-eni** [comment]
Im Build 6.24.2802 wir dar "Neues Objekt" Button immer angezeigt. Klickt der Anwender drauf und keine Thema ist ausgewählt, kommt ein Hinweis:

![image](https://github.com/gis-eni/webgis-forum/assets/62870568/85b97960-adf2-4998-bb67-e8092cebbb98)
